contracts/transfer/circuits/src/dusk_contract/send_to_contract_transparent.rs

```diff
@@ -0,0 +1,304 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use dusk_plonk::constraint_system::ecc::scalar_mul::fixed_base::scalar_mul;
+use dusk_plonk::prelude::*;
+use crate::gadgets::{range::range, commitment::commitment};
+use dusk_plonk::jubjub::{
+    AffinePoint, GENERATOR_EXTENDED, GENERATOR_NUMS_EXTENDED 
+};
+use plonk_gadgets::AllocatedScalar;
+use anyhow::{Error, Result};
+
+/// The circuit responsible for creating a zero-knowledge proof
+/// for a 'send to contract transparent' transaction.
+#[derive(Debug, Default, Clone)]
+pub struct SendToContractTransparentCircuit {
+    /// Value within Pedersen commitment
+    pub commitment_value: Option<BlsScalar>,
+    /// Blinder within Pedersen commitment
+    pub blinder: Option<BlsScalar>,
+    /// Pedersen Commitment 
+    pub commitment: Option<AffinePoint>,
+    /// Value to be sent
+    pub value: Option<BlsScalar>,
+    // Returns circuit size
+    pub size: usize,
+    // Gives Public Inputs
```

Comment:
Make sure the comments for `pi_constructor` and `size` are doc comments.

Reply:
Done.